components/checkout/OrderReview/OrderReview.tsx

```diff
@@ -177,46 +178,52 @@ const OrderReview = (props: OrderReviewProps) => {
               <Typography variant="body1">{shippingPhoneHome}</Typography>
             </Box>
           </OrderInfoHeader>
+      */}
+          {billingAddress?.address1 !== shippingAddress?.address1 &&
+            billingAddress?.address2 !== shippingAddress?.address2 &&
+            billingAddress?.cityOrTown !== shippingAddress?.cityOrTown &&
+            billingAddress?.stateOrProvince !== shippingAddress?.stateOrProvince &&
```

Comment:
Why do we have so many checks? See, if we can refactor it.

Reply:
billingAddress and shipping address is object we cannot check one field over here we need to verify all the fields so if shipping and billing address is same then only billing address will be display